.buildkite/scripts/steps/functional/performance_playwright.sh

```diff
@@ -39,8 +39,13 @@ if [ "$BUILDKITE_PIPELINE_SLUG" == "kibana-performance-data-set-extraction" ]; t
   node scripts/run_performance.js --kibana-install-dir "$KIBANA_BUILD_LOCATION" --skip-warmup
 else
   # pipeline should use bare metal static worker
-  echo "--- Running performance tests"
-  node scripts/run_performance.js --kibana-install-dir "$KIBANA_BUILD_LOCATION"
+  if [[ -z "${JOURNEYS_GROUP+x}" ]]; then
```

Comment:
The `+x` is checking whether a variable is set at all, regardless of weather it has a specific value or empty string. 

Now I wonder if we want to pass empty string to the script: it will handle it and return clear error message, but I also can do it directly in bash. Thoughts?

Reply:
Yeah so currently, the env var can be an empty string, just like you said.
So, the conditional expression could be: `if [[ "${JOURNEYS_GROUP}" = "" ]]; then`

Reply:
In the truth block, you'd return the clear error message

Reply:
I can go with the current solution.

`if [[ "${JOURNEYS_GROUP}" = "" ]]` could break if the variable is missing

Reply:
@wayneseymour - when we run our scripts in strict mode, it will fail with an error on undefined variables. See: 
```
➜  elastic-kibana git:(main) ✗ cat test.sh
#!/usr/bin/env bash

set -euo pipefail

echo "This is a missing variable: $TOMATO"
➜  elastic-kibana git:(main) ✗ bash test.sh
test.sh: line 5: TOMATO: unbound variable
```